code/modules/mob/living/living_health_procs.dm

```diff
@@ -525,7 +525,8 @@
 	hallucination = 0
 	jitteriness = 0
 	dizziness = 0
-
+	stamina.current_stamina = 100
```

Comment:
`stamina.apply_damage(-stamina.max_stamina)` would work better I think, since that makes it update its effects immediately instead of having to wait for a `process()` cycle.
```suggestion
	stamina.apply_damage(-stamina.max_stamina)
```

Reply:
Suggestion applied. thanks 👍